plugins/common/simple_strptime.hpp

```diff
@@ -0,0 +1,11 @@
+#ifndef SIMPLE_STRPTIME_INCL
+#define SIMPLE_STRPTIME_INCL
+
+#include <time.h>
+#include "platform.h"
+
+extern "C" {
```

Comment:
If this does remain compiled into both shared objects (see other comment), there isn't any need for this to be extern "C" and it should also have DECL_LOCAL on the definition to prevent it being exported.

Reply:
Might also be a good idea to use a namespace to ensure that the two copies are regarded as different by tools like debuggers etc.